### PR TITLE
[NEW] This module adds the payment mode on due list of pending payments

### PR DESCRIPTION
**Pull request by cubells**

### CODE REVIEW

**Comment by angelmoya** [comment]
Fallan los test por las dependencias hay que añadir el repo https://github.com/OCA/bank-payment a oca_dependencies.txt

**Comment by cubells** [comment]
@amoya-dx añadido.

Gracias!

**Comment by pedrobaeza** [comment]
Please, talk in english here, as this is an international repo.

File: account_due_list_payment_mode/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Due List Payment Mode
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst:7]: Put the same line above (to highlight the title)

File: account_due_list_payment_mode/README.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* the *Payment Mode* of invoices on account move lines.
+
+Installation
+============
+
+This module depends on :
+* account_due_list
+* account_payment_partner
+
+This module is part of the OCA/bank-payment suite.
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst:20]: Put this sentence next to _account_payment_partner_ module to know which one is.

File: account_due_list_payment_mode/README.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+There is nothing to configure.
+
+Usage
+=====
+
+You are able to list or filter all the due list of pending payments by payment mode.
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst:36]: Add Try me on runbot button: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: account_due_list_payment_mode/README.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+ * No known issues.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/bank-payment/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/bank-payment/issues/new?body=module:%20account_payment_partner%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst]: Change the repository and module here

File: account_due_list_payment_mode/README.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Known issues / Roadmap
+======================
+
+ * No known issues.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/bank-payment/issues>`_.
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/README.rst]: Change repository

File: account_due_list_payment_mode/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+##############################################################################
+
+{
+    "name": "Payment due list with payment mode",
+    "version": "0.1",
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/__openerp__.py]: Use version 8.0.1.0.0 (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)

File: account_due_list_payment_mode/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    "category": "Generic Modules/Payment",
+    "description": """
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/__openerp__.py]: Remove description key as it is in README.rst file.

File: account_due_list_payment_mode/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    This module adds the payment mode to the due list of pending payments.
+    """,
+    "author": "Obertix, Free Solutions",
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/__openerp__.py]: Include "Odoo Community Association (OCA)"

File: account_due_list_payment_mode/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    "website": "http://www.obertix.net",
+    "depends": [
+        "account_payment_partner",
+        "account_due_list",
+    ],
+    "data": [
+        'views/payment_view.xml',
+    ],
+    "installable": True,
+    "auto_install": False,
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/__openerp__.py]: Maybe it can be auto_install...

File: account_due_list_payment_mode/oca_dependencies.txt [hunk 10]
```diff
@@ -0,0 +1,3 @@
+# List the OdooMRP project dependencies, one per line
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/oca_dependencies.txt]: Remove the word OdooMRP (I know from where you have copied it :wink:)

**Comment by cubells** [comment]
@pedrobaeza  I think all is done.

**Comment by pedrobaeza** [comment]
oca_dependencies.txt files should be on root folder, not inside the module folder.

**Comment by cubells** [comment]
Oops. Sorry. I've put the file outside.

**Comment by pedrobaeza** [comment]
Tested on runbot and review technically :+1: 

Thanks

**Comment by angelmoya** [comment]
Tested on runbot :+1:

**Pull request merged**